.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+[Add general information on the proposed Pull Request:
+describe benefits, changes, etc]
+
+
+## Related issue
+[Specify the issue this PR is meant to address]
+
+
+## Checklist
+[Make sure this PR provides:]
```

Comment:
[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
- [ ] [PEP8 compliance](https://www.python.org/dev/peps/pep-0008/)
- [ ] Docstrings for all functions
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Unit tests